tests/Checker/Checks/I18n_Usage_Check_Tests.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Tests for the I18n_Usage_Check_Tests class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Checks\I18n_Usage_Check;
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class I18n_Usage_Check_Tests extends WP_UnitTestCase {
+
+	protected $i18n_usage_check;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->i18n_usage_check = new I18n_Usage_Check();
+	}
+
+	/**
+	 * @covers I18n_Usage_Check::get_args()
+	 */
+	public function test_get_args() {
+
+		$sniffs = $this->i18n_usage_check->get_args();
+
+		$this->assertEquals(
+			array(
+				'extensions' => 'php',
+				'standard'   => 'WordPress,WordPress-Core,WordPress-Docs,WordPress-Extra',
+				'sniffs'     => 'WordPress.WP.I18n',
+			),
+			$sniffs
+		);
+	}
+
+	/**
+	 * @covers I18n_Usage_Check::run()
+	 */
+	public function test_run_with_errors() {
+
+		$check_context = new Check_Context( WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . 'tests/test-plugin-with-errors/test-plugin-with-errors.php' );
```

Comment:
@vishalkakadiya can we not use the `TESTS_PLUGIN_DIR` constant set in the tests bootstrap file? This will remove the need to include the plugin file in the test bootstrap file too.

```suggestion
		$check_context = new Check_Context( TESTS_PLUGIN_DIR . 'tests/test-plugin-with-errors/test-plugin-with-errors.php' );
```

Reply:
Updated this now, thank you!